vthunder/resource_vthunder_slb_health_gateway_test.go

```diff
@@ -0,0 +1,33 @@
+package vthunder
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+var TEST_HEALTH_GATEWAY_RESOURCE = `
+resource "vthunder_slb_health_gateway" "health_gateway1" {
+	sampling_enable {
+		counters1 = "all"
+	}
+}
+`
+
+//Acceptance test
+func TestAccVthunderHealthGateway_create(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
```

Comment:
Is pre Check needed?

Reply:
No, it is just an additional check, not compulsary.